src/Raven.Studio/typescript/viewmodels/database/identities/identities.ts

```diff
@@ -135,11 +135,17 @@ class identities extends viewModelBase {
     private fetchIdentitySeparator() {
         const serverClientConfigurationTask = new getGlobalClientConfigurationCommand()
             .execute()
-            .done(dto => this.serverIdentitySepartor(dto.Disabled ? null : dto.IdentityPartsSeparator || identity.defaultIdentitySeparator));
+            .done(dto => {
+                const separator = dto ? (dto.Disabled ? null : dto.IdentityPartsSeparator || identity.defaultIdentitySeparator) : null;
```

Comment:
this logic becomes tangled
how about:
```
const globalSeparator = (!dto || dto.Disabled) ? null : dto.IdentityPartsSeparator;
this.serverIdentitySeparator(globalSeparator || identity.defaultIdentitySeparator)
```
?

Reply:
sure - done

Reply:
added parenthesis